util.js

```diff
@@ -28,41 +28,27 @@ debugwarn.log = console.warn.bind(console)
 /** @function */
 const isBinaryFileAsync = module.exports.isBinaryFileAsync = Promise.promisify(require('isbinaryfile'))
 
-function removePassword (input) {
+/** @function */
+const removePassword = function (input) {
   return input.replace(/(-P |pass:|\/p|-pass )([^ ]+)/, function (match, p1, p2) {
     return `${p1}***`
   })
 }
 
 /** @function */
 module.exports.execFileAsync = function (file, args, options) {
-  if (debuglog.enabled) {
-    debuglog(`Executing ${file} ${args == null ? '' : removePassword(args.join(' '))}`)
-  }
+  debuglog('Executing...', file, args && Array.isArray(args) ? removePassword(args.join(' ')) : '')
```

Comment:
As we do here string concatenation, it is better to not do it if debug is not enabled. Ok — it doesn't affect performance noticeably :)